### DISCUSSION TITLE
how to uninstall from debian

### DISCUSSION THREAD

**Discussion by nauqi-1** [pear-devs/pear-desktop#2472] [Q&A] [answered]
how would one go about uninstalling this app from debian? since it doesn't seem to have an uninstall script.

**Comment by Lockszmith-GH** [comment]
The question is how did you install it in the first place?

Most likely you've downloaded the AppImage, if that's the case, all you need to do is delete the AppImage

**Reply by nauqi-1** [reply]
yep forgot to mention that. unfortunately i installed it via the .deb file, which i've deleted after installing. :cry: 

if there's no "one-click" way to uninstall it, i'm fine with just deleting any folders that stores the app

**Reply by JellyBrick** [reply, marked as answer]
> yep forgot to mention that. unfortunately i installed it via the .deb file, which i've deleted after installing. 😢
> 
> if there's no "one-click" way to uninstall it, i'm fine with just deleting any folders that stores the app

@nauqi-1 
```bash
sudo apt purge youtube-music
```
